### DISCUSSION TITLE
How to make NvDash transparent?

### DISCUSSION THREAD

**Discussion by joshdumoulin** [NvChad/NvChad#2099] [Q&A] [answered]
Been trying for a while but I just can't seem to figure this out.

**Comment by siduck** [comment]
@joshdumoulin did you try transparency option in chadrc as true?

**Reply by joshdumoulin** [reply]
If you mean the following it still doesn't make transparent nvdash for me.

<img width="510" alt="image" src="https://github.com/NvChad/NvChad/assets/120112283/44ab62f3-792f-4ad3-8826-17f111e10ca7">


<img width="394" alt="image" src="https://github.com/NvChad/NvChad/assets/120112283/90b4af2f-3b4d-4641-9ca8-15d5bd152b5a">

**Reply by siduck** [reply, marked as answer]
@joshdumoulin 

```lua
M.ui = {
  hl_override ={
    NvDashAscii = {
      bg ="none",
      fg ="blue"
    },
    NvDashButtons ={
      bg ="none",
      fg ="light_grey"
    }
  }
}

**Reply by joshdumoulin** [reply]
Thank you very much! :)
